src/Talis/Persona/Client/Base.php

```diff
@@ -55,40 +57,52 @@
     private $defaultTtl;
 
     /**
+     * @var string
+     */
+    private $appUserAgent;
+
+    /**
+     * @var string
+     */
+    private $phpVersion;
+
+    /**
      * Constructor
      *
      * @param array $config An array of options with the following keys: <pre>
      *      persona_host: (string) the persona host you'll be making requests to (e.g. 'http://localhost')
      *      persona_oauth_route: (string) the token api route to query ( e.g: '/oauth/tokens')
+     *      userAgent: Consuming application user agent string @since 2.0.0
+     *            examples: rl/5.2, rl, rl/5, rl/5.2 (php/5.3; linux/2.5)
```

Comment:
Ultimately, it's completely up to the client what to send here.  The docs are just trying to establish best practices.  IMO, tarl sending the release it's running is more useful than anything else I can think of after the slash.  That way we know if some behavior began with a specific release.